app/presenters/queries/content_item_warnings.rb

```diff
@@ -0,0 +1,23 @@
+module Presenters
+  module Queries
+    module ContentItemWarnings
+      def self.call(content_id, state, base_path, document_type)
+        if state == "draft"
+          blocking_content_item_id = ::Queries::CheckForContentItemPreventingDraftFromBeingPublished.call(
+            content_id,
+            base_path,
+            document_type,
+          )
+
+          if blocking_content_item_id.nil?
+            nil
+          else
+            {
+              "content_item_blocking_publish" => "There is an item of content that prevents this draft from being published"
+            }
+          end
```

Comment:
could you replace the if/else with an `unless blocking_content_item_id.nil?` - (as the method will return `nil` by default)